### PR TITLE
#10 Create account

### PR DESCRIPTION
**Pull request by RemusCM**
#10 Finished basic functionality. Need time guards for credit card and line of credit, Also need to contact Stan to help us connect to actual server (I used localhost). Finally, need registration/login of client complete so we can use the specific client_id, I used a hard-coded id.

### CODE REVIEW

**Comment by jaslatendresse** [comment]
I removed myself from reviewers since i worked on that branch.

File: classes/Account.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ */
+
+class Account
+{
+    private $account_number;
+    private $balance;
+    private $account_type;
+    private $option_name;
+
+    public function __construct($account_number, $balance, $account_type, $option_name)
```
Comment (samantha-kerr) [inline classes/Account.php]: Account is missing 'level' and 'interest rate'. Also, 'option' is only relevant to Saving and Checking accounts.

File: classes/Credit.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ */
+
+class Credit extends Account
+{
+    private $credit_limit;
+
+    public function __construct($account_number, $balance, $account_type, $option_name, $credit_limit)
+    {
+        parent::__construct($account_number, $balance, $account_type, $option_name);
+        $this->credit_limit = $credit_limit;
```
Comment (samantha-kerr) [inline classes/Credit.php]: Need minimum payment attribute too

**Comment by samantha-kerr** [comment]
@jaslatendresse  @RemusCM  all good here?

**Comment by jaslatendresse** [comment]
@samantha-kerr let me just change the db connection to the school one and we good

**Comment by jaslatendresse** [comment]
@samantha-kerr done. however, i can't test anything for now as my local server won't connect to my local db...but last time i tested it everything was good

**Comment by samantha-kerr** [comment]
I'll merge and try it on mine.

**Pull request merged**